### ISSUE TITLE
Improve meetup listing API

### ISSUE THREAD

**Issue by c3o** [hack4vdb/anthill-backend#17]
/api/meetupsnearactivist/:
- Enforce same limits as in the staticviews: Max x events, max y km away, sorted by date & must be in the future (pls make x & y centrally configurable)
- Add generated event proposals (Tosh wants the 3 times as 3 separate events)

**Comment by PeterTheOne** [comment]
1. Done in https://github.com/hack4vdb/anthill-backend/commit/96ee0b2fcb9cd7745de7f77942cbb22d012b6d76

**Comment by PeterTheOne** [comment]
2: some version of this is located here `api/interestingplaces/<user_uuid>`. I will fix this.

**Comment by c3o** [comment]
Note: Once this is done, "join_meetup_bot" (misnamed) will also need to be able to create events, not just users

**Comment by PeterTheOne** [comment]
2: is done in https://github.com/hack4vdb/anthill-backend/commit/bf4d582ecec554c1c0f79cb89b8e1a14501360da

**Comment by PeterTheOne** [comment]
@c3o open separate issue for join_meetup_bot, if you haven't already
